### PR TITLE
Add support for Freetrade

### PR DESCRIPTION
**Pull request by lildude**
## Added

- Support for [Freetrade](https://freetrade.io/)

As an aside, this is a smaller part of a [bigger PR](https://github.com/lildude/Export-To-Ghostfolio/pull/1) I've been working on in my fork which resolves a few issues and niggles I encountered when trying to add support for Freetrade. I know things like rearranging a repo can step on toes so I've left this PR to be solely about adding Freetrade as the repo stands.

If you like all the changes in my other PR and are happy to accept them, I'm happy to close this PR in favour of that one, or split it up into a number of smaller easier to review PRs.

### CODE REVIEW

**Comment by dickwolff** [comment]
Thanks for the PR (both actually). I will look into this in more detail soon (currently planning on Tuesday)!

File: src/converters/freetradeConverter.test.ts [hunk 1]
```diff
@@ -0,0 +1,164 @@
+import { FreetradeConverter } from "./freetradeConverter";
+import { YahooFinanceService } from "../yahooFinanceService";
+import { GhostfolioExport } from "../models/ghostfolioExport";
+
+describe("freetradeConverter", () => {
+
+  beforeEach(() => {
+    jest.spyOn(console, "log").mockImplementation(jest.fn());
```
Comment (dickwolff) [inline src/converters/freetradeConverter.test.ts:8]: I saw this change in your other PR, and want to take a better look at this. Can you remove the mock from this PR so it works like the other converter tests for now?

File: src/converters/freetradeConverter.ts [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+                        currency: record.instrumentCurrency,
+                        dataSource: "YAHOO",
+                        date: dayjs(record.dividendPayDate).format("YYYY-MM-DDTHH:mm:ssZ"),
+                        symbol: security.symbol
+                    });
+
+                    bar1.increment();
+                    continue;
+                }
+                // Buy & Sell
```
Comment (dickwolff) [inline src/converters/freetradeConverter.ts:157]: Can you add a white line between these?

**Comment by dickwolff** [comment]
Overall a good PR, a few small things I left in review for you to look at. 

I also took a look at your other PR. I'm not opposed to do a repository shake-up. The current setup was a merge of two separate converters and it has become somewhat cluttery (regarding the sample files). I want to propose to do both PR's separatly.  This way this new feature can be added first, and then I can review the reorganisation separatly. I do have some questions about that PR, so I'll add them there.

**Review by dickwolff — COMMENTED** [review]
Can you bump the GitVersion file to 0.9.0?

Also, a test is [failing](https://github.com/dickwolff/Export-To-Ghostfolio/actions/runs/8859565200/job/24442450465?pr=55), looks like an import thing.

**Comment by dickwolff** [comment]
Merged! Thanks for the contribution!

I added comments to your draft PR, feel free to open it as a merge to this repo!

**Comment by lildude** [comment]
🤔 I'm not seeing your comments

**Comment by lildude** [comment]
Hold off. I'm going to open a new PR as I can't rebase back to this repo.

**Comment by dickwolff** [comment]
> 🤔 I'm not seeing your comments

Did you manage to find them?

**Comment by lildude** [comment]
Ah, you've not finished the review - note the "Pending". You need to click the "Review changes" button on the "Files changed" tab. I won't be able to see review comments until then.

**Comment by dickwolff** [comment]
I have added them manually to the new PR.

**Comment by lildude** [comment]
Still not seeing them 😬

**Pull request merged**